keps/sig-api-machinery/4192-svm-in-tree/README.md

```diff
@@ -203,6 +250,9 @@ When the Migrator controller is integrated in-tree, it will leverage the `Stream
 - _Open Question_:
     - Does `Streaming List` support inconsistent lists? Currently, with chunked lists, we receive inconsistent lists. We need to determine if we can achieve the same with Streaming List. Depending on the outcome, we may consider removing the [ContinueToken](https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/60dee538334c2366994c2323c0db5db8ab4d2838/pkg/apis/migration/v1alpha1/types.go#L63) from the API.
 
+#### Garbage Collection Cache
```

Comment:
The above subsection above streaming lists - I suggest moving it to "rejected alternatives" or sth like that.
The current plan of record is to not use it even after Alpha.